<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressDialog</class>
 <widget class="QDialog" name="ProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>操作进度</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#lblStatus {
    color: #2c3e50;
    font-weight: bold;
}
QProgressBar {
    height: 20px;
    text-align: center;
}
QProgressBar::chunk {
    background-color: #3498db;
    width: 10px;
    margin: 0.5px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lblOperation">
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
font-weight: bold;
color: #34495e;</string>
     </property>
     <property name="text">
      <string>当前操作：准备开始</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="format">
      <string>%p%</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatus">
     <property name="text">
      <string>状态：初始化中...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>操作日志</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="styleSheet">
         <string notr="true">font-family: 'Consolas', 'Courier New', monospace;
font-size: 10pt;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>操作日志将显示在这里...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    min-width: 80px;
    padding: 5px;
    background-color: #e74c3c;
    color: white;
    border-radius: 4px;
}
QPushButton:hover {
    background-color: #c0392b;
}</string>
       </property>
       <property name="text">
        <string>取消操作</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icons/cancel.png</normaloff>:/icons/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>ProgressDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
